src/net.cpp

```diff
@@ -1871,7 +1871,7 @@ void CConnman::ThreadMessageHandler()
             // Send messages
             {
                 LOCK(pnode->cs_sendProcessing);
-                GetNodeSignals().SendMessages(pnode, *this, flagInterruptMsgProc);
+                GetNodeSignals().SendMessages(pnode, *this);
```

Comment:
Even if we dont use it now, I'm not sure we want to get rid of passing flagInterruptMsgProc through...there is a strong case for using it in 0.15.